docs/reference/commandline/run.md

```diff
@@ -134,89 +134,82 @@ previous changes intact using `docker start`. See `docker ps -a` to view a list
-    none            1.9G     0  1.9G   0% /mnt
+```bash
+$ docker run -d -p 8080:80 nginx
+```
 
-The `--privileged` flag gives *all* capabilities to the container, and it also
-lifts all the limitations enforced by the `device` cgroup controller. In other
-words, the container can then do almost everything that the host can do. This
-flag exists to allow special use-cases, like running Docker within Docker.
+### More complex examples
 
-### Set working directory (-w)
 
-    $ docker  run -w /path/to/dir/ -i -t  ubuntu pwd
+#### Full container capabilities (--privileged)
 
-The `-w` lets the command being executed inside directory given, here
-`/path/to/dir/`. If the path does not exist it is created inside the container.
+```bash
+$ docker run --rm --it ubuntu bash
+root@bc338942ef20:/# mount -t tmpfs none /mnt
+mount: permission denied
+```
 
-### Set storage driver options per container
+Trying to run the `mount` command from within an unprivileged container results in
+an error, because by default, most potentially dangerous kernel capabilities are
+dropped, including `cap_sys_admin` (which is required to mount filesystems).
+Running the same command inside a privileged container started with the
+`--privileged` flag allows the `mount` command to succeed.
 
-    $ docker run -it --storage-opt size=120G fedora /bin/bash
+```bash
+$ docker run -it --privileged ubuntu bash
+root@50e3f57e16e6:/# mount -t tmpfs none /mnt
+root@50e3f57e16e6:/# df -h
+Filesystem      Size  Used Avail Use% Mounted on
+none            1.9G     0  1.9G   0% /mnt
+```
 
-This (size) will allow to set the container rootfs size to 120G at creation time.
-User cannot pass a size less than the Default BaseFS Size. This option is only
-available for the `devicemapper`, `btrfs`, `windowsfilter`, and `zfs` graph drivers.
+The `--privileged` flag gives *all* capabilities to the container and lifts all
+the limitations enforced by the `device` cgroup controller. A privileged container
+can do almost anything that the host machine can do. You must also use privileged
+containers to run a Docker container within another Docker container. However,
+privileged containers introduce an additional level of risk to the host machine
+and other containers.
 
-### Mount tmpfs (--tmpfs)
+#### Mount tmpfs (--tmpfs)
 
     $ docker run -d --tmpfs /run:rw,noexec,nosuid,size=65536k my_image
 
 The `--tmpfs` flag mounts an empty tmpfs into the container with the `rw`,
 `noexec`, `nosuid`, `size=65536k` options.
 
-### Mount volume (-v, --read-only)
-
-    $ docker  run  -v `pwd`:`pwd` -w `pwd` -i -t  ubuntu pwd
-
-The `-v` flag mounts the current working directory into the container. The `-w`
-lets the command being executed inside the current working directory, by
-changing into the directory to the value returned by `pwd`. So this
-combination executes the command using the container, but inside the
-current working directory.
+#### Mount volume (-v, --read-only)
```

Comment:
For a follow-up; looks like we only describe _bind mounting_ host directories here, and not actual _volumes_